develop/interoperability/intro-to-xcm.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Introduction to XCM
+description: Unlock seamless blockchain interoperability with XCM—Polkadot's cross-consensus messaging format for asset transfers and cross-chain interactions.
+---
+
+# Introduction to XCM
+
+## Introduction
+
+Polkadot’s unique value lies in its ability to enable interoperability between parachains and other blockchain systems. At the core of this capability is XCM (Cross-Consensus Messaging)—a flexible messaging format that facilitates communication and collaboration between independent consensus systems.
+
+With XCM, blockchains can exchange assets, data, and instructions seamlessly, fostering a more interconnected ecosystem. Although it was developed specifically for Polkadot, XCM is a universal format, adaptable to any blockchain environment. This guide provides an overview of XCM’s core principles, design, and functionality, alongside practical examples of its implementation.
+
+## Messaging Format
+
+XCM is not a protocol but a standardized [messaging format](https://github.com/polkadot-fellows/xcm-format){target=\_blank}. It defines the structure and behavior of messages but does not handle their delivery. This separation allows developers to focus on crafting precise instructions for target systems without worrying about transmission mechanics.
+
+XCM messages are action-oriented, specifying how the receiving blockchain should alter its state. Unlike transactions, they do not directly perform these changes. By leveraging asynchronous composability, XCM enables efficient, out-of-sequence execution, akin to how RESTful services structure HTTP requests.
```

Comment:
What do you mean by this? Out-of-sequence?
Also XCMs just have intents, the underlying changes are decided by the host environment (the chains).

Reply:
agree, will paraphrase this paragraph just a bit to align it with francisco's comment :)